core-war/src/main/java/org/silverpeas/web/directory/servlets/DirectoryRequestRouter.java

```diff
@@ -147,6 +147,11 @@ public String getDestination(String function, DirectorySessionController directo
         users = directorySC.getLastListOfUsersCalled();
         destination = doPagination(request, users, directorySC);
 
+      } else if (function.equalsIgnoreCase("ChangeNumberItemsPerPage")) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Move the "ChangeNumberItemsPerPage" string literal on the left side of this string comparison. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://integration.silverpeas.org/sonar/coding_rules#rule_key=squid%3AS1132)